README.md

```diff
@@ -265,6 +265,12 @@ the server and show `info`-level log messages and above:
 ```
 $ RUST_LOG=info cargo run server
 ```
+Set the
+`RUST_BACKTRACE` environment variable in order to turn on full rust tracktrace. For example, run
+the server abd turn on debugging and full backtrace:
```

Comment:
`the server abd turn on debugging and full backtrace:` should be `the server and turn on debugging and full backtrace:`